docs/Diagnostics.md

```diff
@@ -90,6 +90,25 @@ The Swift compiler has a setting (under LangOptions) called `DiagnosticsEditorMo
 
 Most diagnostics have no reason to change behavior under editor mode. An example of an exception is the "protocol requirements not satisfied diagnostic"; on the command line, it may be better to show all unsatisfied requirements, while in an IDE a single multi-line fix-it would be preferred.
 
+### Educational Notes ###
+
+**Note**: This feature is currently experimental. It can be enabled by passing the `-Xfrontend -enable-descriptive-diagnostics`.
```

Comment:
Nit: Either "passing the `-Xfrontend -enable-descriptive-diagnostics` flag" or just "passing `-Xfrontend -enable-descriptive-diagnostics`".

Reply:
good catch, thanks!